pkg/infra/remotecache/redis_storage.go

```diff
@@ -93,7 +94,15 @@ func (s *redisStorage) Set(ctx context.Context, key string, data []byte, expires
 
 // GetByteArray returns the value as byte array
 func (s *redisStorage) Get(ctx context.Context, key string) ([]byte, error) {
-	return s.c.Get(ctx, key).Bytes()
+	item, err := s.c.Get(ctx, key).Bytes()
+	if err != nil {
+		if errors.Is(err, redis.Nil) {
+			return nil, ErrCacheItemNotFound
+		}
+		return nil, err
+	}
+
+	return item, nil
```

Comment:
Nice catch

Reply:
we should have enforced handled errors in our linter no?

Reply:
This just fix an inconsistency where all other backing storages for remote caching was returning `ErrCacheItemNotFound` and the _redis_ storage return `redis.Nil`. It still returned an error but a different one